### PR TITLE
Add test generator for prime factors exercise

### PR DESCRIPTION
**Pull request by robkeim**
Another one for #195

### CODE REVIEW

File: generators/Exercises/PrimeFactors.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Generators.Exercises
+{
+    public class PrimeFactors : Exercise
```
Comment (ErikSchierboom) [inline generators/Exercises/PrimeFactors.cs]: I love it when the generator can look like this 😎

Comment (robkeim) [inline generators/Exercises/PrimeFactors.cs]: Me too 👍

**Comment by jpreese** [comment]
Eek, this PR removed all of the skips in the test suite.

**Comment by robkeim** [comment]
Good catch @jpreese, thanks for fixing this! I'm working in Visual Studio which doesn't allow me to ignore the skips so I have to remove them to validate our sample solution is working. I forgot to re-run the generator in this case.

**Pull request merged**